### ISSUE TITLE
Set up lint rules

### ISSUE THREAD

**Issue by nicks101** [infiniteoverflow/Libro#9]
Adding linting at the beginning of the project would help maintain good practices moving forward.
  
There are several options to add lint :   
 
1.  Add rules individually as you go.
2. Use a community package from pub dev.  

I prefer to use the second option. There are several packages available.   
I would prefer [lint](https://pub.dev/packages/lint).  (No particular reason)

It is highly customizable. We can remove or change the label of any rule.

**Comment by infiniteoverflow** [comment]
Yes sure , you can go ahead with lint package. Its the best we can use for linting. I will assign you this issue once the coding period starts. Good luck !!

**Comment by nicks101** [comment]
Okay, on it.

**Comment by infiniteoverflow** [comment]
@nicks101 You can get started with this.
